### DISCUSSION TITLE
Read a image and write on disk,image size changed?

### DISCUSSION THREAD

**Discussion by jsxzliyou** [deepjavalibrary/djl#1335] [Q&A] [answered]
`Path imageFile = Paths.get("D:/1.jpg");
Image img = ImageFactory.getInstance().fromFile(imageFile);
Path imagePath = Paths.get("D:/2.jpg");
img.save(Files.newOutputStream(imagePath), "jpg");
`
2.jpg size less then 1.jpg

**Comment by frankfliu** [comment, marked as answer]
@jsxzliyou 
This is expected, a couple reason can cause this happend:
1. The original file contains metadata (e.g. IDV3Tags)
2. The original file has a different color scheme than saved file (BufferredImageFactory save jpg file as TYPE_INT_ARGB format)
3. jpeg uses a lossy compression algorithm, every saving may cause the image content change.
